pkg/envvars/envvars_test.go

```diff
@@ -19,9 +19,9 @@ func TestBuild(t *testing.T) {
 		{
 			name: "should create envvars without prefix",
 			expected: map[string]string{
-				"STATUS_LISTENERS_0_TYPE":             "secure",
-				"STATUS_LISTENERS_0_ADDRESSES_0_HOST": "my-cluster-kafka-bootstrap.coffeeshop.svc",
-				"STATUS_LISTENERS_0_ADDRESSES_0_PORT": "9093",
+				"status_listeners_0_type":             "secure",
+				"status_listeners_0_addresses_0_host": "my-cluster-kafka-bootstrap.coffeeshop.svc",
+				"status_listeners_0_addresses_0_port": "9093",
```

Comment:
is this change still required?

Reply:
yes,now we don't apply uppercasing by default when we prepare list of binding names.